rats-tls/src/include/rats-tls/api.h

```diff
@@ -111,15 +111,15 @@ typedef struct rtls_evidence {
 #define RATS_TLS_API_VERSION_MAX	RATS_TLS_API_VERSION_1
 #define RATS_TLS_API_VERSION_DEFAULT RATS_TLS_API_VERSION_1
```

Comment:
There seems to be an unalignment here on row 112 originally. It is not caused by your changed.

Reply:
I'll submit another PR to fix it.